src/data/Pager/PagerTheme.ts

```diff
@@ -28,6 +29,7 @@ const baseTheme: PagerTheme = {
   },
   ellipsis: 'cursor-pointer',
   pagerDisplayItems: 'mr-1.5',
+  visibleRange: '',
```

Comment:
This name isn't the most obvious - What about `showPageCount` or something ?

Reply:
<img width="182" alt="Screenshot 2024-07-18 at 18 03 36" src="https://github.com/user-attachments/assets/6dc9f5a4-0b82-4259-9c52-0c21b754f242">

`showPageCount` it not suitable, need to define only visible (shown) items

maybe `shownItems` will be better.
What do you think?

Reply:
how about - `showPageRange`

Reply:
good for me, I'll update